### PR TITLE
Updated 2.5 strings.yaml

### PR DESCRIPTION
**Pull request by chrisrooneys**
I rewrote the file to hopefully make more sense. I will be online early so we can chat live if needed.

### CODE REVIEW

File: translations/strings.yaml [hunk 1]
```diff
@@ -86,4 +91,144 @@ splittunnel:
...
+  dnsModalBodyText: Custom DNS servers allow for faster speed using local networks. You can find this feature in the “Network settings” section.
+  dnsModalButtonText: Done
+  tourPageHeader: What’s new
+  tourSubHeader: Take the tour
+  multiListItemHeader: Multi-hop VPN
+  multiListItemBody: Multi-hop VPN uses multiple VPN servers instead of one for extra security and privacy.
+  supportListItemHeader: In-app support form
+  supportListItemBody: The in-app support form will streamline the process of getting your issues resolved.
+  dnsListItemHeader: Custom DNS
+  dnsListItemBody: Custom DNS settings allow for faster connection using local servers.
```
Comment (bakulf) [inline translations/strings.yaml:210]: replace local with custom?

**Review by bakulf — APPROVED** [review]

File: translations/strings.yaml [hunk 2]
```diff
@@ -86,4 +91,144 @@ splittunnel:
...
+  supportFieldHeader: How can we help you with Mozilla VPN?
+  dropdownLabel: 
+    value:
+      - Choose a category
+    comment:
+      - A user trying to submit feedback will use this dropdown to select the general topic related to the feedback.
+  paymentDropdownListItem: 
+    value:
+      - Payment and billing
+      comment:
```
Comment (bakulf) [inline translations/strings.yaml:131]: identation. Otherwise the yaml file is invalid :/

File: translations/strings.yaml [hunk 3]
```diff
@@ -86,4 +91,144 @@ splittunnel:
...
+      - A user selecting this will be submitting in-app feedback related to payment and billing.
+  accountDropdownListItem: 
+    value:
+      - Account issues
+     comment:
```
Comment (bakulf) [inline translations/strings.yaml:136]: same here

File: translations/strings.yaml [hunk 4]
```diff
@@ -86,4 +91,144 @@ splittunnel:
...
+
+# IAP Android edge case 3
+genericPurchaseError:
+  genericPurchaseErrorHeader: Problem confirming subscription
+  genericPurchaseErrorText: An unexpected error has occurred, please try again. Visit our help center below to learn how to troubleshoot this issue or contact support.
+  genericPurchaseErrorButton: Try Again
+
+# v2.5 What’s New? Release notes
+whatsNewReleaseNotes:
+  comment:
```
Comment (bakulf) [inline translations/strings.yaml:189]: we do not support comments for categories

**Review by bakulf — CHANGES_REQUESTED** [review]

File: translations/strings.yaml [hunk 5]
```diff
@@ -69,7 +74,7 @@ settingsDev:
   useStagingSubtitle: Staging services are used to test features before they go live to production
   restartRequired: VPN must be restarted to take effect
 
-splittunnel:
+splitTunnel:
```
Comment (flodolo) [inline translations/strings.yaml]: This is going to break existing code? Strings are not exposed yet, so it's not an issue for l10n.

**Review by flodolo — APPROVED** [review]
I'm OK with merging this and iterating. I'll keep string extraction on pause in the meantime

File: translations/strings.yaml (hunk 5 above)
Comment (bakulf) [reply translations/strings.yaml]: This is fixed by my commit.

**Pull request merged**